c2rust-analyze/src/rewrite/expr/mir_op.rs

```diff
@@ -166,11 +170,11 @@ impl<'a, 'tcx> ExprRewriteVisitor<'a, 'tcx> {
                             if rv_place.is_indirect()
                                 && self.acx.local_tys[rv_place.local].ty.is_any_ptr()
                             {
-                                let local_addr = self.acx.local_tys[rv_place.local].label;
-                                let perms = self.perms[local_addr];
-                                let flags = self.flags[local_addr];
-                                if let (Ownership::Cell, _) = type_desc::perms_to_desc(perms, flags)
-                                {
+                                let local_lty = self.acx.local_tys[rv_place.local];
```

Comment:
i see why you wrapped this up into a variable to pass `local_lty.ty` to `perms_to_desc` but I think this is a step down in clarity for understanding that the `label` is the `local_addr`

Reply:
The previous version sometimes used `local_addr` for the `addr_of_local` `PointerId` and other times used it for the outermost `PointerId` of the local's type (`local_tys[local].label`).  I changed it so that `local_addr` is used exclusively for the former.

In this module, we're pretty consistent about using the name `lty` only for things of type `crate::LTy`, so `some_lty.label` is always a `PointerId`.  I could name it as `local_ptr` or something, though, if you think that would help.

Reply:
i think it would help people like me, primarily because i'm not keeping in mind what module i'm in when looking at `LTy` and so `label` is something I'd have to jump to to know what it is. There is at least `RwLty` which makes it more easily guessable and so `.label` variable names don't necessarily need the same massaging to communicate meaning

Reply:
OK, added `local_ptr` vars in a3c72909